### PR TITLE
update submitfilter for phanpy and golett

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
@stdweird, @wdpypere  please review

File: bin/submitfilter.py [hunk 1]
```diff
@@ -86,7 +86,18 @@
         'TOTMEM': 1395439 << 20,  # total amount of ram in dugtrio cluster
         'DEFMAXNP': 48,  # default maximum np in case of ppn=full
         },
-    }
+
```
Comment (stdweird) [inline bin/submitfilter.py]: remove or add everywhere

File: bin/submitfilter.py [hunk 2]
```diff
@@ -86,7 +86,18 @@
...
+    'phanpy': {
+        'PHYSMEM': 528271884 << 10,
```
Comment (stdweird) [inline bin/submitfilter.py]: this needs to be adjusted for the increased pagepool. (add a comment why it's different then the output of pbsnodes)

Comment (JensTimmerman) [inline bin/submitfilter.py]: what number does it need to be then? I copied this from pbsscripts...

Comment (stdweird) [inline bin/submitfilter.py]: i forgot to update it there, that's why i mention it here.
it's a 16GB pagepool, dunno what the old value was, but it's 16GB less :smile:

**Comment by JensTimmerman** [comment]
please rereview

File: bin/submitfilter.py [hunk 3]
```diff
@@ -224,7 +234,8 @@ def main(arguments=sys.argv):
         maxvmem = 0
         if opts.ppn in ('all', 'half'):
             opts.ppn = 1
-        sys.stderr.write("Warning: unknown server (%s) detected, see PBS_DEFAULT. This should not be happening...\n" % opts.server)
+        sys.stderr.write("Warning: unknown server (%s) detected, see"
```
Comment (stdweird) [inline bin/submitfilter.py]: does this work? multiple lines should be wrapped in `()` themself, no?

Comment (stdweird) [inline bin/submitfilter.py]: apparently not. seems to work

**Pull request merged**